commercetools-java-client-core/src/test/java/io/sphere/sdk/client/TokensSupplierTest.java

```diff
@@ -68,7 +68,7 @@ public void shouldSendUserAgent() throws Exception {
         final Optional<String> userAgentHeader = httpRequest.getHeaders().findFlatHeader(HttpHeaders.USER_AGENT);
         assertThat(userAgentHeader).isPresent();
         final String userAgent = userAgentHeader.get();
-        assertThat(userAgent).startsWith("commercetools-jvm-sdk/");
+        assertThat(userAgent).startsWith(UserAgentUtils.userAgent);
```

Comment:
actually in this case it's better to use the string here. Cause so you are testing the variable is used, but not the content of the variable

Reply:
Yes, make sense :) I put it anyway as a constant in the class test itself just to avoid to have some many places to change it in case we need to change it again ;)